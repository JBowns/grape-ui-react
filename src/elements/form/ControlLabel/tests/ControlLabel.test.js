import React from 'react';
import { ControlLabel } from '..';
import 'jest-styled-components';
import renderer from 'react-test-renderer';

describe('ControlLabel Component base', () => {
  it('should return a ControlLabel object', () => {
    const component = renderer.create(<ControlLabel htmlFor="forWhatId">Label text is here</ControlLabel>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
