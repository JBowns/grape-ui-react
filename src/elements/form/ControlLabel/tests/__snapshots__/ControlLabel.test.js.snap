// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ControlLabel Component base should return a ControlLabel object 1`] = `
.c0 {
  background-color: hsl(0,0%,100%);
  color: inherit;
  font-family: inherit;
  font-size: 0.8rem;
  font-weight: inherit;
  -webkit-letter-spacing: inherit;
  -moz-letter-spacing: inherit;
  -ms-letter-spacing: inherit;
  letter-spacing: inherit;
  line-height: 1.5;
  text-align: inherit;
  -webkit-text-decoration: inherit;
  text-decoration: inherit;
  left: 0.5rem;
  padding: 0 0.5rem;
  position: absolute;
  top: -0.5rem;
}

<label
  className="c0"
  htmlFor="forWhatId"
>
  Label text is here
</label>
`;
